src/components/Header.tsx

```diff
@@ -42,13 +42,15 @@ const MenuLink = ({
   return (
     <RouteLink
       href={href}
-      color="white"
+      color="primaryTextLightmode"
       borderBottomWidth={isActive ? '4px' : 'none'}
```

Comment:
This is what we had before for the `borderBottomWidth`. I think both options are OK for now. So I'll let you pick the one you like best. 😉